src/routes/jira/sync/jira-sync-post.ts

```diff
@@ -9,8 +9,12 @@ import { booleanFlag, BooleanFlags } from "~/src/config/feature-flags";
 export const JiraSyncPost = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const { installationId: gitHubInstallationId, appId: gitHubAppId } = req.body;
 
+	if (!req.body?.commitsFromDate) {
+		res.status(404).send("Missing sync date");
```

Comment:
`400` instead of `404`?

Reply:
Again made it optional. There are few scenarios where this API is being called without `commitFromDate`